tools/ocean_neural_network/canyonb.xml

```diff
@@ -0,0 +1,91 @@
+<tool id="bgc_canyon_b" name="Canyon-B" version="@VERSION@">
+    <description>Robust Estimation of Open Ocean CO2 Variables and Nutrient Concentrations From T, S, and O2 Data Using Bayesian Neural Networks</description>
+    <macros>
+        <token name="@VERSION@">0.9.3</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="4.3.2">r-base</requirement>
+        <requirement type="package" version="@VERSION@">r-canyonb</requirement>
+    </requirements>
+    <required_files>
+        <include type="literal" path="canyonb.r"/>
+    </required_files>
+    <command detect_errors="exit_code"><![CDATA[ 
+        Rscript
+            '$__tool_directory__/canyonb.r'             
+            '$date'
+            '$latitude'
+            '$longitude'
+            '$pressure'
+            '$temperature'
+            '$salinity'
+            '$dissoxygen'
+            '$variables'
+            '$epressure'
+            '$etemperature'
+            '$epsalinity'
+            '$edissoxygen'
+            '$output'
+        ]]>
+    </command>
+    <inputs>
+        <param name="date" type="text" value="yyyy-mm-dd HH:MM" label="date (UTC) as string ('yyyy-mm-dd HH:MM')"/>
```

Comment:
This is date and time isn't it?

Reply:
What about a special date selector in Galaxy? And a time selector in addition?

Reply:
It does not exist yet, but we could create one for next year if you think it could be useful for many tools and many commuities.

Reply:
Aaah yes it would be very useful. I think almost one out of 2 of the tools I added this last 2 years needed a date.

Reply:
Ok, can you please add a comment here. The more detailed the better. E.g. sketching out use-cases. How the UI should look like, but also how the "string" should look like (could be multiple ones) that you get on the command section.